### PR TITLE
Dispatcher refactor

### PR DESCRIPTION
**Pull request by kenwheeler**
- Switching from global dispatcher to Biff instance specific dispatcher
  instance
- Updating tests & build

cc @per-nilsson

### CODE REVIEW

**Comment by kenwheeler** [comment]
Also @per-nilsson , see proposed async action syntax here: 

https://github.com/kenwheeler/mcfly/issues/37

File: lib/Action.js [hunk 1]
```diff
@@ -12,8 +11,9 @@ var invariant = require('invariant');
    * @param {function} callback - Callback method for Action
    * @constructor
    */
-  function Action(callback) {"use strict";
+  function Action(callback,dispatcher) {"use strict";
```
Comment (coopy) [inline lib/Action.js]: global: add space after comma, please

**Comment by coopy** [comment]
After style inconsistency fixes, LGTM. I'm going to check out the [FB panel talk](https://github.com/kenwheeler/mcfly/issues/36#issuecomment-76000986) about async in actions vs stores and see what I think makes most sense for our use cases for Biff.

**Comment by coopy** [comment]
LGTM :+1:  :rocket: :shipit:

**Pull request merged**